active-directory-dotnet-native-uwp-v2/MainPage.xaml.cs

```diff
@@ -75,7 +75,7 @@ private async void CallGraphButton_Click(object sender, RoutedEventArgs e)
 
                 try
                 {
-                    authResult = await PublicClientApp.AcquireTokenInteractive(scopes, this)
+                    authResult = await PublicClientApp.AcquireTokenInteractive(scopes)
```

Comment:
Don't we want to add .WithParentActivityOrWindow?

Reply:
There isn't any on UWP.